filters/filters-2023.txt

```diff
@@ -3081,3 +3081,13 @@ bisnis.adigenius.com##img[id*="wait"]
 
 ! https://github.com/AdguardTeam/AdguardFilters/issues/154228
 filmestorrent.tv##+js(noeval-if, adsBlocked)
+
+! aoutoqw.xyz skip countdown
+||www.google.com/recaptcha/api.js$script,domain=aoutoqw.xyz,important
+aoutoqw.xyz##+js(nano-sib)
+aoutoqw.xyz##b:has-text(Open Link)
+aoutoqw.xyz##center:has(> a:has(> img))
```

Comment:
Does

```adb
aoutoqw.xyz##center > a > img:upward(center)
```

work for you? It would be more efficient than using using nested `has`.

Reply:
still working :D

Reply:
Does the site have many `b` or `center` elements? It would be better if you can restrict them before using `has-text`

Reply:
If center are rare then for Chromium 105+ we can use CSS4:
```adb
aoutoqw.xyz##center:has(> a > img)
```

Reply:
I have refactored that to make more efficient.

```
aoutoqw.xyz##b:has-text(Open Link)
aoutoqw.xyz##center:has(> a)
```

here the element i want to remove:

![ss](https://github.com/uBlockOrigin/uAssets/assets/136988406/4a66fce8-9265-44df-b4a0-968a0a4bb5f1)
```html
<html>
  <head>
    <link
      rel="stylesheet"
      href="//cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
    />
    <script
      src="https://www.google.com/recaptcha/api.js"
      async=""
      defer=""
    ></script>
  </head>
  <body>
    <form
      method="post"
      action="https://aoutoqw.xyz/how-to-become-a-pro-player-in-mobile-legends-and-various-hero-heroes.html"
      target=""
      id="alf_continue_form"
    >
      <div
        class="g-recaptcha"
        data-sitekey="6LdEisUZAAAAAJpV9-1dgaOVtkmXyJ_n0sCFVyPV"
        data-callback="enableContinueBtn"
      ></div>
      <br />
      <br /><br />
      <center>
        <b><a href="//waufooke.com/4/5874463">Open Link</a></b
        ><br />
        <center xv85pvfa2="" icwv474o0="">
          <a href="//sotchoum.com/4/5938776" target="_blank" rel="noopener"
            ><img
              class="alignnone size-full wp-image-163"
              src="https://aoutoqw.xyz/wp-content/uploads/2023/06/bannerputih.jpg"
              alt=""
              width="350"
              height="450"
          /></a>
        </center>

        <div id="alf_continue_button_outer">
          <input type="hidden" name="alf_get_link" value="Y" />
          <button
            id="alf_continue_captcha"
            class="alf_button btn btn-secondary"
          >
            Continue
          </button>
        </div>
        <center></center>
        <style>
          .rc-anchor-checkbox-label {
            font-size: 23px !important;
            font-weight: 600 !important;
            line-height: 37px !important;
          }
          .g-recaptcha {
            display: inline-block;
          }
          #alf_continue_captcha {
            cursor: not-allowed;
            opacity: 0.5;
          }
          #alf_continue_form {
            width: 100%;
            margin-top: 40px !important;
            text-align: center;
            float: none;
            margin: 0 auto;
          }
        </style>

        <script>
          function enableContinueBtn() {
            document.getElementById('alf_continue_captcha').disabled = false;
            document.getElementById('alf_continue_captcha').style.cursor =
              'pointer';
            document.getElementById('alf_continue_captcha').style.opacity = '1';
          }
        </script>
      </center>
    </form>
  </body>
</html>
```

Reply:
here my final solution.

```
||google.com/recaptcha/api.js$script,domain=aoutoqw.xyz,important
aoutoqw.xyz,~go.aoutoqw.xyz##+js(nano-sib)
aoutoqw.xyz##b
aoutoqw.xyz##center > a
aoutoqw.xyz##.wpadcenter-ad-container
aoutoqw.xyz###alf_continue_captcha:remove-attr(disabled)
aoutoqw.xyz###alf_continue_captcha:style(cursor: pointer !important; opacity: 1 !important;)
```